app/(app)/(purchase-and-payment)/purchase/index.tsx

```diff
@@ -28,12 +27,19 @@ export type PurchaseSearchParams = {
   udrId?: string
 }
 
+interface IPriceError {
```

Comment:
We prefer types to interfaces. Use interface only if type cannot be used.
https://bratislava.github.io/frontend/components

Reply:
Is PurchaseScreen the right place to store this type/interface?